packages/grafana-ui/src/components/VizLayout/VizLayout.tsx

```diff
@@ -5,6 +5,7 @@ import { useMeasure } from 'react-use';
 import { GrafanaTheme2 } from '@grafana/data';
 import { LegendPlacement } from '@grafana/schema';
 
+import { config } from '../../../../../public/app/core/config';
```

Comment:
@simPod thanks for contribution.

Instead of importing the config , please use `cont theme = useTheme2();`  and then get the break point value like this: `theme.breakpoints.values.lg.`

Other than that this looks good.